### ISSUE TITLE
Ability to show unit information for debugging

### ISSUE THREAD

**Issue by brentm5** [drapergeek/luckycharms#21]
It would be nice to be able to show a summary of the unit on the screen.  For example you want to find a particular unit and its in a large area.  You set the unit to "locate" mode and it will show a custom screen with information about the unit on it.

**Comment by drapergeek** [comment]
This is a really interesting idea. So basically we would need a custom view that the unit will then display as its webpage.

:+1: I LIKE IT

**Comment by brentm5** [comment]
Yes that's the idea. I don't know the best way to go about doing this. Maybe just a local file. That is created by the unit ?

**Comment by drapergeek** [comment]
What do you mean?  I'm thinking we have something that is just a controller action on the server that will display this info as long as you pass an API key

**Comment by brentm5** [comment]
That's exactly what I mean .... Now
